docs/governance.md

```diff
@@ -0,0 +1,140 @@
+# Overview
+
+An overview of the technical side of Composable's governance.
+
+This document describes entities' and origins' ability to exercise authority rather than enacting token economics motions. Configuration values for the protocol are not set in stone by governance  (and can be generalized to other protocols as well). Instead, it allows for decision-making and provides a sandbox for what governance can be.
```

Comment:
I don't think this sentence makes any sense tbh (or may be you can reword)
```suggestion
```

Reply:
indeed, haha. it happend after several changes by several people:)

Reply:
reworded closer to original